cmd/crc-embedder/cmd/embed.go

```diff
@@ -90,7 +90,7 @@ var (
 			hyperkit.MachineDriverDownloadURL,
 			hyperkit.HyperkitDownloadURL,
 			constants.GetOcURLForOs("darwin"),
-			constants.GetCrcTrayDownloadURL(),
+			constants.GetCRCMacTrayDownloadURL(),
```

Comment:
`CRC` could be dropped from the name, `constants.GetMacTrayDownloadURL` is already clear